src/components/TestEndpoints.php

```diff
@@ -36,6 +37,18 @@ private static function checkForwardedHeader(): void
         self::respondJson('{}');
     }
 
+    private static function checkDnsResolve(): void
+    {
+        $ip = gethostbyname('www.google.com');
+        if ($ip !== 'www.google.com') {
```

Comment:
```suggestion
        if ($ip === 'www.google.com') {
```

Maybe, the condition should be reversed?